world/petitions/migrations/0004_auto_20181219_1343.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-12-19 13:43
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('character', '0033_goal_goalupdate'),
+        ('dominion', '0039_prestigetier'),
+        ('petitions', '0003_brokeredsale_broker_type'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Interest',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=30)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='WantedAd',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('playtime_start', models.PositiveIntegerField(default=0)),
+                ('playtime_end', models.PositiveIntegerField(default=0)),
+                ('prefer_weekend', models.BooleanField(default=False)),
+                ('blurb', models.TextField()),
+                ('marriage', models.BooleanField(default=False)),
+                ('sponsor', models.BooleanField(default=False)),
+                ('protege', models.BooleanField(default=False)),
+                ('plot', models.BooleanField(default=False)),
+                ('active', models.BooleanField(default=False)),
+                ('clues', models.ManyToManyField(to='character.Clue')),
+                ('interests', models.ManyToManyField(to='petitions.Interest')),
+                ('owner', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='wanted_ad', to='dominion.PlayerOrNpc')),
+                ('searchtags', models.ManyToManyField(to='character.SearchTag')),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.AddField(
+            model_name='interest',
+            name='wantedads',
+            field=models.ManyToManyField(related_name='wanted_ads', to='petitions.WantedAd'),
+        ),
+    ]
```

Comment:
This or your other PR will need to be changed when one or the other is merged in, because the petitions will have shared dependencies.

Reply:
I only created the migration because Travis can't run without it. I think it probably makes the most sense for you to handle the migrations on your end.